multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/steps/ValidateDeployParametersStep.java

```diff
@@ -144,6 +144,10 @@ private void verifyArchiveSignature(ProcessContext context, Path archiveFilePath
         if (!context.getVariable(Variables.VERIFY_ARCHIVE_SIGNATURE)) {
             return;
         }
+
+        if (!configuration.isArchiveSignatureVerificationEnabled()) {
```

Comment:
This will evaluate to true if the feature is _disabled_, not the other way around.
If I understand correctly, we _want_ it to be disabled.

Reply:
No, it is checking if it is enabled. The current logic is working